### PR TITLE
[GST-1327] Fixes kafka settings so ports are set up correctly

### PR DESCRIPTION
**Pull request by lobsterdore**
PR

### CODE REVIEW

File: templates/server.properties.j2 [hunk 1]
```diff
@@ -1,27 +1,21 @@
 ##
 # {{ ansible_managed }}
 
-##
-# Server Basics
+############################# Server Basics #############################
```
Comment (dm) [inline templates/server.properties.j2]: Why? :stuck_out_tongue_closed_eyes:

File: templates/server.properties.j2 [hunk 2]
```diff
@@ -1,27 +1,21 @@
...
-# The port the socket server listens on
-port={{ url.split(':')[1] }}
-  {% endif %}
-{% endfor %}
+port={{ kafka.port }}
 
 # Hostname the broker will bind to. If not set, the server will bind to all interfaces
 {% if kafka.listen_address is defined %}
 host.name={{ kafka.listen_address }}
-{% else %}
```
Comment (dm) [inline templates/server.properties.j2]: You shouldn't remove the `{% else %}` if there's no `kafka.listen_address` defined in `main.yml`

File: templates/server.properties.j2 (hunk 1 above)
Comment (lobsterdore) [inline templates/server.properties.j2]: It matches the format in the rest of the file.

Comment (dm) [inline templates/server.properties.j2]: I'd prefer the opposite... What's the standard we're following for these?

File: templates/server.properties.j2 (hunk 2 above)
Comment (lobsterdore) [inline templates/server.properties.j2]: It's not needed, the else line was commented out, and Kafka will listen on localhost by default.

Comment (dm) [inline templates/server.properties.j2]: :ok_hand:

File: templates/server.properties.j2 (hunk 1 above)
Comment (lobsterdore) [inline templates/server.properties.j2]: I really don't mind either way to be honest, I can update the other headers if you want me to?

Comment (dm) [inline templates/server.properties.j2]: Oh I'm not precious about it either way as well... Feel free to keep it as is.
Was just asking for consistency if it's one way or the other :smile:

**Comment by dm** [comment]
LGTM :+1: 

LGTM

**Pull request merged**